app/models/manageiq/providers/ibm_power_hmc/infra_manager/lpar.rb

```diff
@@ -34,6 +34,13 @@ def raw_rename(new_name)
     end
   end
 
+  def do_request(request_type, options)
+    case request_type
+    when 'clone_to_template'
```

Comment:
If this is the only possible request type can we just call `make_template` directly?  Or are you skipping other request types?